backend.native/tests/interop/objc/msg_send/messaging.h

```diff
@@ -0,0 +1,47 @@
+#import <objc/NSObject.h>
+
+@interface PrimitiveTestSubject : NSObject
+
++ (int)intFn;
++ (float)floatFn;
++ (double)doubleFn;
+
+@end;
+
+typedef struct {
+    float f;
+} SingleFloat;
+
+typedef struct __attribute__((packed)) {
+    char f1;
+    short f2;
+    char f3;
+    char f4;
+} SimplePacked;
+
+typedef struct {
+    float f1;
+    float f2;
+    float f3;
+    float f4;
+} HomogeneousSmall;
+
+typedef struct {
+    float f1;
+    float f2;
+    float f3;
+    float f4;
+    float f5;
+    float f6;
+    float f7;
+    float f8;
+} HomogeneousBig;
```

Comment:
SIMD types?

Reply:
We don't have support for them yet. What do you meat?

Reply:
I just meant, that we need to check for types like int8x16_t in those tests eventually. Another possible kind is bitfields, like `unsigned foo:2`.